ws-server/src/server.rs

```diff
@@ -638,6 +642,7 @@ impl Default for Settings {
 	fn default() -> Self {
 		Self {
 			max_request_body_size: TEN_MB_SIZE_BYTES,
```

Comment:
If we agree to name this `max_reponse_size` then we should rename `max_request_body_size` to `max_request_size`

Reply:
I renamed `max_response_size` to `max_response_body_size`  and kept the `max_request_body_size` unchanged for code compatibility.